verifier/src/qe_identity.rs

```diff
@@ -646,6 +677,7 @@ mod test {
         let time = "2023-07-12T20:48:25Z"
             .parse::<DateTime>()
             .expect("Failed to parse time");
+        let time = Some(time);
```

Comment:
I have a lot of these in the test code. Not sure how much it matters, but it might look confusing reassigning time immediately. I like this better than the alternative because the linter forces you to add a weird comma to the end of `Some` blocks.

Reply:
that linter is odd, I would probably do it in the function call, but IFF the `Into<Option` path is chosen they could all go away too.
```rust
let verifier = SignedQeIdentityVerifier::new(Some(key), Some(time));
```